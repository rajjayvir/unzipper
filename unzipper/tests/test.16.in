0x0a
0x68 0x10 0x00
0x67 0x28 0x00
0x66 0x3c 0x00
0x65 0x4e 0x00
0x64 0x5f 0x00
0x0a 0x6f 0x80
0x78 0x9f 0xc0
0x61 0x9f 0xc8
0x62 0x8f 0xd0
0x63 0x7f 0xe0
0x00 0x00 0x02 0x07
0xfc 0x7e 0x7f 0x3f 0xbf 0xbf 0xbf 0xbf 0xff 0xff 
0xff 0xff 0xff 0xff 0xfe 0xf7 0xbd 0xef 0x7b 0xde 
0xf7 0xbd 0xef 0x7b 0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 
0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 0xdd 
0xdf 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6d 0xb6 0xdb 0x7d 0x55 0x55 0x55 0x55 
0x55 0x55 0x55 0x55 0x55 0x55 0x55 0x55 0x55 0x55 
0x55 0x55 0xf5 0x55 0x55 0x55 0x55 0x55 0x55 0x55 
0x55 0x55 0x55 0x55 0x55 0x55 0x55 0x55 0x57 0xc0 
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x1f 0x00 0x00 
0x00 0x00 0x00 0x00 0x00 0x00 0x7c 0x00 0x00 0x00 
0x00 0x00 0x00 0x00 0x01 0xf0 0x00 0x00 0x00 0x00 
0x00 0x00 0x00 0x07 0xc0
