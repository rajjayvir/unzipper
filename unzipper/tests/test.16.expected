xaabbbbccccccccddddddddddddddddeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
